src/views/Notifications/Home/_components/_utils.tsx

```diff
@@ -291,3 +291,22 @@ export const pushValuesToChangesArray = (
   pushValues(obj);
   return changesArray;
 };
+
+/**
+ * __extractContent__
+ *
+ * `metaData.content` has html elements in it, causing difficulty to render it onto page without extra massaging. This function parses out the raw text.
+ *
+ * Example:
+
+* - from: ```<p><span data-type=\"mention\" tag-type=\"USER_ACCOUNT\" class=\"mention\" data-id=\"JTTC\" data-id-db=\"735bee9e-90b2-4bf7-95f8-9039bbdc03b3\" data-label=\"Anabelle Jerde (JTTC)\">@Anabelle Jerde (JTTC)</span> reply</p>```
+ *
+ * - to: `@Anabelle Jerde (JTTC) reply`
+ *
+ * @param {string} metaData.content
+ * @returns string
+ */
+export const extractContent = (text: string) => {
+  return new DOMParser().parseFromString(text, 'text/html').documentElement
+    .textContent;
+};
```

Comment:
I'm a little unsure of this approach, as we intentionally pass back the entire mention element to be able to maintain certain features that we may want to leverage in the future, such as hover text, links to user account, etc.  This approach could work if we rebuild the element completely, but as is, we are losing metadata that we may want to retain.  I think it might be worth exploring adding a class over this element to fix a stying issue, rather than extracting the text.